<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X31ScActiveForm</class>
 <widget class="QWidget" name="X31ScActiveForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image:none;</string>
       </property>
       <property name="text">
        <string>主动宽带警戒</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MFramepColorPlot" name="activePlot" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image:none;</string>
       </property>
       <property name="text">
        <string>主动搜索信息</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="tbAcSearch"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MFramepColorPlot</class>
   <extends>QWidget</extends>
   <header>mframepcolorplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
